Dockerfile.cypress

```diff
@@ -0,0 +1,20 @@
+# base image
+FROM cypress/browsers:chrome65-ff57
+
+# set working directory
+RUN mkdir /usr/src/app
+WORKDIR /usr/src/app
+
+COPY package*.json ./
```

Comment:
Things I learned to explain better: In the Dockerfile, Docker is good at caching each step upon subsequent rebuilds after changing say some of your JS files but things such as the node_modules dependencies do not change. It's often best practice to put the command that are least likely to change and can be cached first like installing the node_modules and then the ones that are more likely to change after those commands.

Reply:
This resource helped to explain the caching aspects of the Dockerfile steps and why we do it in our Dockerfiles too as the order matters: https://medium.com/ai2-blog/dockerizing-a-react-application-3563688a2378